README.md

```diff
@@ -57,11 +57,8 @@ If adding a camera which does not needed mapped because the focus point is given
 
 Known Issues
 --------
-1. Not currently working on Windows. 
-2. Lightroom has a bug where lrPhoto:getDevelopSettings()["Orientation"] always returns nil. Lightroom does not
```

Comment:
Looks good. Let's address some of the comments. I'll pull it into a feature branch and for QA. After it passes, I'll merge it into master